files/job_wrapper.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+
+# Replicate input files directory structure
+# @TODO: This could be executed 
```

Comment:
We have:

```
etc/job_wrapper.sh
files/job_wrapper.sh
```

It is not clear which files are for what.

What about using:

```
etc/htcondorcern/job_wrapper.sh
etc/htcondorvc3/job_wrapper.sh
```

so that we can easily support several backends and distinguish between them?

This would mean that we'd have to move quite a few CERN-specific files from `etc/*` to /etc/htcondorcern/*` ourselves... but merging this branch will be quite a lot of work anyway, so could perhaps do it alongside?!

(An alternative would be to use the same directory but strictly "cern" and "vc" naming prefix everywhere.)

Reply:
I like creating different subdirectories in etc:
```
etc/htcondorvc3/job_wrapper.sh
etc/htcondorcern/job_wrapper.sh
```
As you said, handling several different backends would be easier to manage and visualize than prefixes or suffixes (`job_wrapper_cern.sh / cern_job_wrapper.sh`).

If you're OK with the subdirectories in `etc/`, I'd be happy to help move CERN specific files as well as the VC3 files.